docs/modules/publisher-directmethods.md

```diff
@@ -280,37 +280,39 @@ Returns the nodes configured for one Endpoint (Dataset)
   >
   > ```json
   > {
-  >   "EndpointUrl":"opc.tcp://192.168.100.20:50000"
+  >    "EndpointUrl": "opc.tcp://192.168.100.20:50000"
   > }
   > ```
   >
   > _Response_:
   >
   > ```json
   > {
-  >   "status":200,
-  >   "payload":[
-  >     {
-  >       "id":"nsu=http://microsoft.com/Opc/OpcPlc/;s=SlowUInt1",
-  >       "opcSamplingInterval":3000,
-  >       "opcSamplingIntervalTimespan":"00:00:03",
-  >       "heartbeatInterval":0,
-  >       "heartbeatIntervalTimespan":"00:00:00"
-  >     }
-  >   ]
+  >    "status": 200,
+  >    "payload": [
```

Comment:
Payload should now open an object not a list.

Reply:
done